testing/buildbots.rst

```diff
@@ -16,7 +16,7 @@ will schedule a new build to be run as soon as possible.
 
 The build steps run by the buildbots are the following:
 
-* Check out the source tree for the changeset which triggered the build
+* Check out the source tree for the pull request which triggered the build
```

Comment:
GitHub workflow builds might be triggered by both a PR creation event, or by individual commits.  I'm not too familiar with buildbots though, so I'm not sure if "pull request" is correct here or if it should be "commit" (or both).

Reply:
I'll change it to "change."